content/download/ubuntu.adoc

```diff
@@ -47,13 +47,12 @@ sudo add-apt-repository --yes ppa:js-reynaud/ppa-kicad
 sudo apt-get update
 sudo apt-get install kicad
 
-=== Old Stable
-The 2013 stable release of KiCad is currently in the official Ubuntu repository.
-It is not recommended for new projects. Please use the new stable release.
-To install old stable search for `KiCad` in the Ubuntu Software Center or execute the following in a terminal:
+=== Stable Release in the Ubuntu repository
+The stable release of KiCad 4.0.2 is currently in the official Ubuntu repository.
+To install it search for `KiCad` in the Ubuntu Software or execute the following in a terminal:
 
 [source,bash]
-sudo apt-get install kicad
+sudo apt install kicad kicad-common
```

Comment:
Why remove the `-get`?

I don't think we need to specify `kicad-common` explicitly because that is a dependency of kicad specified in the package.

I think we should still mention that the old stable is not recommended, because 4.0.2 is only available in 16.04 and up, I am sure some people are still using old ubuntus, so I would like to keep the warning.